mlops/parallelm/mlops/ml_metrics_stat/auc.py

```diff
@@ -0,0 +1,33 @@
+from parallelm.mlops.metrics_constants import ClassificationMetrics
+from parallelm.mlops.mlops_exception import MLOpsStatisticsException
+from parallelm.mlops.stats.single_value import SingleValue
+from parallelm.mlops.stats_category import StatCategory
+
+
+class AUC(object):
```

Comment:
This class will be removed (since it is repeating itself for most of the stats.. 
The cleanup will come in the 13 PR after the inital 12 metric goes in.

Reply:
cleanup/beautiy pr: https://github.com/mlpiper/mlpiper/pull/93